Remora.Discord.Commands/Attributes/DiscordDefaultDMPermissionAttribute.cs

```diff
@@ -25,25 +25,22 @@
 namespace Remora.Discord.Commands.Attributes;
 
 /// <summary>
-/// Marks a group with a default permission, that is, whether commands in the group will be visible to all users by
-/// default.
+/// Marks a command group as being either executable or non-executable in a DM by default.
```

Comment:
Update the documentation to reflect that the attribute can be used on both groups and commands.